apply-notch-hax.sh

```diff
@@ -18,7 +34,9 @@ if [ ! -e $DIFF ]; then
        please tune the files on your device manually first,
        then fork the repo, create a patches/$DEVICE.diff
        and modify this script to fetch the patches from your
-       fork for testing!"
+       fork for testing!
+	   Alternatively, specify a device template to use with
```

Comment:
The 2 lines here are indented with a few spaces following a tab while everything else above is only spaces, could you make them match that? Also I thought the new line before this new paragraph (which was in the example I gave before) looked nice, so maybe you could add it as well? :)